### ISSUE TITLE
Upon installing "TypeScript" from Package.io, certain commands don't seem to work

### ISSUE THREAD

**Issue by fionafung** [Microsoft/TypeScript-Sublime-Plugin#137]
While I get syntax coloring and object identifier intellisense, commands such as go to def, rename, etc don't seem to work, and the intellisense list doesn't appear to be funded by TS.

**Comment by DanielRosenwasser** [comment]
Same problem. It started out as a broken pipe error as it was probably starting up, but now here's the error I keep getting in the console:

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 556, in run_
    return self.run(edit)
  File "TypeScript in /Users/Daniel/Library/Application Support/Sublime Text 3/Installed Packages/TypeScript.sublime-package", line 924, in run
  File "TypeScript in /Users/Daniel/Library/Application Support/Sublime Text 3/Installed Packages/TypeScript.sublime-package", line 440, in checkUpdateView
  File "TypeScript in /Users/Daniel/Library/Application Support/Sublime Text 3/Installed Packages/TypeScript.sublime-package", line 421, in reloadBuffer
  File "./codecs.py", line 885, in open
NotADirectoryError: [Errno 20] Not a directory: '/Users/Daniel/Library/Application Support/Sublime Text 3/Installed Packages/TypeScript.sublime-package/.tmpbuf'
```

@fionafung, any chance you could peek at your console and see if you're getting the same error?

**Comment by DanielRosenwasser** [comment]
Also, what is the use of .tmpbuf if I may ask?

**Comment by billti** [comment]
Fixed in 0.1.3 which published.  Should be live shortly. (You may need to restart Sublime after it is installed)
